transports/transports_test.go

```diff
@@ -0,0 +1,21 @@
+package transports
+
+import (
+	"testing"
+
+	"github.com/containers/image/directory"
+	"github.com/containers/image/storage"
+	"github.com/containers/image/transports"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTransportFromImageName(t *testing.T) {
+	dirTransport := transports.TransportFromImageName("dir:/tmp/test")
+	assert.Equal(t, dirTransport.Name(), directory.Transport.Name())
+	storageTransport := transports.TransportFromImageName("containers-storage:ref:test")
```

Comment:
Note that the storage transport is optional (can be compiled out using a build tag); the test should not rely on it.

Reply:
Alright, I dropped that case.